README.md

```diff
@@ -3,7 +3,7 @@
 [![npm version][npm-image]][npm-url]
 [![CI][ci-image]][ci-url]
 
-This CLI tool enforces the following aspects of consistency across a monorepo / yarn workspace:
+This CLI tool enforces the following aspects of consistency across a monorepo / NPM/Yarn workspace:
```

Comment:
How about?

```suggestion
This CLI tool enforces the following aspects of consistency across a monorepo with npm or Yarn workspaces:
```

`A / B/C` reads a bit odd, as if it is a typo. Also npm is canonically written lower case: https://en.wikipedia.org/wiki/Npm_(software)

Reply:
Good suggestion!